library/src/main/java/org/codice/security/saml/EntityInformation.java

```diff
@@ -18,25 +18,18 @@
 import com.google.common.collect.ImmutableSet;
 import java.util.*;
 import java.util.function.Predicate;
-import java.util.stream.Collectors;
 import javax.annotation.concurrent.Immutable;
 import org.apache.commons.lang3.StringUtils;
 import org.codice.security.saml.SamlProtocol.Binding;
 import org.opensaml.core.xml.schema.XSBase64Binary;
 import org.opensaml.saml.saml2.core.AuthnRequest;
-import org.opensaml.saml.saml2.metadata.AssertionConsumerService;
-import org.opensaml.saml.saml2.metadata.Endpoint;
-import org.opensaml.saml.saml2.metadata.EntityDescriptor;
-import org.opensaml.saml.saml2.metadata.KeyDescriptor;
-import org.opensaml.saml.saml2.metadata.SPSSODescriptor;
-import org.opensaml.saml.saml2.metadata.SingleLogoutService;
+import org.opensaml.saml.saml2.metadata.*;
```

Comment:
❓ Are we allowing wildcard imports in java classes?

Reply:
No. Obviously not. I can't believe you even had to ask :trollface: